pageserver/src/tenant/timeline.rs

```diff
@@ -294,6 +294,10 @@ pub struct Timeline {
     /// Completion shared between all timelines loaded during startup; used to delay heavier
     /// background tasks until some logical sizes have been calculated.
     initial_logical_size_attempt: Mutex<Option<completion::Completion>>,
+
+    /// Load or creation time information about the disk_consistent_lsn and when the loading
+    /// happened. Used for consumption metrics.
+    pub(crate) loaded_at: (Lsn, SystemTime),
```

Comment:
TenantHarness sadly allows creating only "fresh" timelines. Need to make sure that the real ones will have access to disk_consistent_lsn from the metadata...

Reply:
Ok, it seems the local testing required no changes except testability.